tests/test_permissions.py

```diff
@@ -90,14 +135,12 @@ def test_is_site_admin_user(self, username, allow):
         ('super_user', True),
         ('superstaff_user', True),
         ('alpha_nonadmin', False),
-        ('alpha_site_admin', True),
+        ('alpha_site_admin', False),
         ('nosite_staff', False),
     ])
     def test_multiple_user_orgs(self, username, allow):
         """
-        Allow users to have multiple orgs.
-
-        Probably we'll remove this features, but this ensures Tahoe doesn't go crazy.
+        Prevent users from having multiple orgs.
```

Comment:
@OmarIthawi Can we test that `Organization.MultipleObjectsReturned` is raised for this test?:

https://github.com/appsembler/course-access-groups/blob/omar/multi-orgs/course_access_groups/permissions.py#L58

I'll raise my broader concern in the main PR dialog

Reply:
I believe it is tested: 

https://github.com/appsembler/course-access-groups/blob/68d84edc79f72b244fcec46f0a653e185c891181/tests/test_permissions.py#L50-L59